README.md

```diff
@@ -4,18 +4,7 @@ The CENNZnet JavaScript API library for browsers and Node.js.
 
 ## Quick Start
 
-You must use **yarn** with @cennznet/api and set the following **resolutions** in your package.json, otherwise your install may break due to breaking changes in downstream package versions.
-``` 
-"resolutions": {
-  "@polkadot/types": "1.2.1",
-  "@polkadot/metadata": "1.2.1",
-  "@polkadot/api": "1.2.1",
-  "@polkadot/api-derive": "1.2.1",
-  "@polkadot/rpc-core": "1.2.1",
-  "@polkadot/rpc-provider": "1.2.1",
-  "@polkadot/jsonrpc": "1.2.1",
-}
-``` 
+You must use **yarn** with @cennznet/api.
```

Comment:
do we still need to enforce this? if so can you add a brief description of why

Reply:
https://github.com/polkadot-js/api#development
To start off, this repo (along with others in the @polkadot family) uses yarn workspaces to organise the code. As such, after cloning, its dependencies should be installed via yarn, not via npm; the latter will result in broken dependencies.

Can we add the same reason?

Reply:
I don't think so, this reason is for developing **on** the @polkadot/* codebases not a requirement for users of their packages

Reply:
ok.. I will remove the enforcement then..